test/PushNotificationTests/APITests.cpp

```diff
@@ -188,72 +197,142 @@ namespace Test::PushNotifications
 
         }
 
+        TEST_METHOD(MultipleBackgroundActivation_Unpackaged)
+        {
+            RunTestUnpackaged(L"BackgroundActivationTest", testWaitTime()); // Need to launch one time to enable background activation.
+
+            auto LocalBackgroundTask1 = winrt::create_instance<winrt::Windows::ApplicationModel::Background::IBackgroundTask>(c_comServerId, CLSCTX_ALL);
+            auto mockBackgroundTaskInstance1 = winrt::make<MockBackgroundTaskInstance>();
+
+            auto LocalBackgroundTask2 = winrt::create_instance<winrt::Windows::ApplicationModel::Background::IBackgroundTask>(c_comServerId, CLSCTX_ALL);
+            auto mockBackgroundTaskInstance2 = winrt::make<MockBackgroundTaskInstance>();
+
+            VERIFY_NO_THROW(LocalBackgroundTask1.Run(mockBackgroundTaskInstance1));
+            VERIFY_NO_THROW(LocalBackgroundTask2.Run(mockBackgroundTaskInstance2));
```

Comment:
Nit: This is the same code as in the equivalent packaged-app test. We probably want to have one common function for this and call it in both tests. It's fine, I won't block the PR because of this.

Reply:
These are unit tests, having two with some duplicated code itsn't a big deal at this time. Plus, we have a deliverable to move as many tests as possible from the test app, into this file. When this happen, there will be significant code churn and refactoring opportunities